packages/pluggableWidgets/range-slider-web/src/RangeSlider.xml

```diff
@@ -0,0 +1,202 @@
+                        <enumerationValue key="static">Static</enumerationValue>
+                        <enumerationValue key="dynamic">Dynamic</enumerationValue>
+                        <enumerationValue key="expression">Expression</enumerationValue>
+                    </enumerationValues>
+                </property>
+                <property key="stepValue" type="decimal" defaultValue="1">
+                    <caption>Step size</caption>
+                    <description />
+                </property>
+                <property key="stepAttribute" type="attribute" required="false">
+                    <caption>Step size</caption>
+                    <description/>
+                    <attributeTypes>
+                        <attributeType name="Integer"/>
+                        <attributeType name="Long"/>
+                        <attributeType name="Decimal"/>
+                    </attributeTypes>
+                </property>
+                <property key="expressionStepSize" type="expression" required="false">
+                    <caption>Step size</caption>
+                    <description />
+                    <returnType type="Decimal" />
+                </property>
+                <!-- Steps Size End -->
+            </propertyGroup>
+            <propertyGroup caption="Tooltip">
+                <property key="showTooltip" type="boolean" defaultValue="true">
+                    <caption>Show tooltip</caption>
+                    <description />
+                </property>
+                <property key="tooltipTypeLower" type="enumeration" defaultValue="value">
+                    <caption>Lower bound tooltip type</caption>
+                    <description />
+                    <enumerationValues>
+                        <enumerationValue key="value">Value</enumerationValue>
+                        <enumerationValue key="customText">Custom</enumerationValue>
+                    </enumerationValues>
+                </property>
+                <property key="tooltipLower" type="textTemplate" required="false">
+                    <caption>Tooltip</caption>
+                    <description />
+                </property>
+                <property key="tooltipTypeUpper" type="enumeration" defaultValue="value">
+                    <caption>Upper bound tooltip type</caption>
+                    <description>By default tooltip shows current value. Choose 'Custom' to create your own template.</description>
+                    <enumerationValues>
+                        <enumerationValue key="value">Value</enumerationValue>
+                        <enumerationValue key="customText">Custom</enumerationValue>
+                    </enumerationValues>
+                </property>
+                <property key="tooltipUpper" type="textTemplate" required="false">
+                    <caption>Tooltip</caption>
+                    <description />
+                </property>
```

Comment:
Yes, this is true, but this division comes from the requirement that you can have "textTemplate" for tooltip. and because each "handle" has own value source, we have to use template for each tooltip individually.

Reply:
I think you're explaining why we have the type attribute and the text template attribute, that's perfectly fine and I understand. My question is more that the design document from Chris didn't specify a tooltip for the lower bound and the upper bound, so I'm asking where this came from? Because AFAIK Slider also only has 1 tooltip for the current value. Did you add this yourself or discuss with Chris? @iobuhov

Reply:
I did it myself. This is just most simple solution to have tooltip for range slider. I expect get feedback on PM review

Reply:
@cjfhodges can you make sure to keep this into account?